src/displayBackend/DisplayBackend.cpp

```diff
@@ -155,7 +156,9 @@ DisplayBackend::DisplayBackend(DisplayPtr display,
 	BackendBase("DisplBackend", deviceName),
 	mDisplay(display)
 {
-	mDisplay->start();
+	assert(mDisplay);
+
+	mDisplay->start();	
```

Comment:
Please, don't leave trailing whitespaces at EOL.

By the way, `git diff` will mark them with red rectangles. This is hard to miss.

Reply:
fixed